vuepress/docs/next/tutorials/create/ms/add-api-claim.md

```diff
@@ -50,7 +50,7 @@ app.listen(port, function () {
     console.log('Listening to Port ' + port);
 }); 
 ```
->REST APIs should be added to microservices on Entando, following REST frameworks.
+>Entando recommends adding REST APIs to microservices.
```

Comment:
@nshaw, needed to add note about adding REST APIs to MSs but it looks out of place here in this tutorial. What do you think? It was hard to find an appropriate place for it. Another alternative is the api-management.md but that's for the ent CLI info and that seemed odd too.